### PR TITLE
Update Viz docs for EveryNSlabs

### PR DESCRIPTION
**Pull request by mar-celine**
## Proposed changes

After #1416, I believe the visualization documentation is outdated. I'll admit I haven't figured out yet how to properly observe using the new changes, so these changes are only a rough draft. I'm hoping for clarification. If the changes required are too extensive, I can open an issue instead and let someone else fill in the correct documentation.

### Types of changes:

- [ ] Bugfix
- [ ] New feature

### Component:

- [ ] Code
- [ ] Documentation
- [ ] Build system
- [ ] Continuous integration

### Code review checklist

- [ ] The PR passes all checks, including unit tests, `clang-tidy` and `IWYU`.
  For instructions on how to perform the CI checks locally refer to the [Dev
  guide on the Travis CI](https://spectre-code.org/travis_guide.html).
- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).

### CODE REVIEW

**Comment by nilsdeppe** [comment]
Marked as a priority since this fixes what I view as a critical defect in #1416

**Comment by wthrowe** [comment]
Can we do YAML snippets?

**Comment by nilsdeppe** [comment]
Ooooh that's an amazing idea @wthrowe!! It would be really great to do that, especially as we start writing more tutorials. I'll have to give it a try to see what Doxygen supports :)

File: docs/Tutorials/Visualization.md [hunk 1]
```diff
@@ -27,7 +27,18 @@ the executable using the flag `--input-file`. For example, for a scalar wave
 evolution, run the command:
 `./EvolveScalarWave3D --input-file Input3DPeriodic.yaml`.
 By default, the example input files do not produce any output. This can be
-changed by modifying the options passed to `ObserveNSlabs` and `ObserveAtT0`.
+changed by modifying the options passed to `EventsAndTriggers`:
+
+```
```
Comment (kidder) [inline docs/Tutorials/Visualization.md:32]:
```yaml
EventsAndTriggers:
  ? <trigger_1>
  : - <event_1>
    - <event_2>
   ...
    - <event_n>
  ? <trigger_2>
  : - <event_1>
    - <event_2>
   ...
    - <event_n>
   ...
  ? <trigger_m>
  : - <event_1>
    - <event_2>
   ...
    - <event_n>
```



```yaml
EventsAndTriggers:
  ? EveryNSlabs:
      N: 3
      Offset: 5
  : - Observe: 
        VariablesToObserve: [MassDensity]
```

File: docs/Tutorials/Visualization.md [hunk 2]
```diff
@@ -27,7 +27,18 @@ the executable using the flag `--input-file`. For example, for a scalar wave
...
+EventsAndTriggers:
+  Observe:
+    VariablesToObserve: [MassDensity]
+    EveryNSlabs:
+      N: 3
+      Offset: 5
+```
```
Comment (mar-celine) [inline docs/Tutorials/Visualization.md:39]: ? key : value, together called a yaml map
A - is used for each item in a list.

**Comment by mar-celine** [comment]
@kidder I believe the snippet

```yaml
EventsAndTriggers:
  ? <trigger_1>
  : - <event_1>
    - <event_2>
   ...
    - <event_n>
  ? <trigger_2>
  : - <event_1>
    - <event_2>
   ...
    - <event_n>
   ...
  ? <trigger_m>
  : - <event_1>
    - <event_2>
   ...
    - <event_n>
```
Should go in its own Events and Triggers documentation. I am willing to make this addition in a separate PR, just to keep this one short and mergeable in a short time frame.

**Comment by wthrowe** [comment]
I tried creating a YAML snippet.  It works, but you have to adjust `EXAMPLE_PATH` in Doxygen.in to tell it to search all of `@PROJECT_SOURCE_DIR@/tests/` instead of just the `Unit` subdirectory.

**Comment by kidder** [comment]
@gsb76 okay; please make an issue for putting that snippet in

**Review by kidder — APPROVED** [review]

**Comment by kidder** [comment]
@wthrowe should the yaml snippet be added now?  I think we can have example input files that are just checked and not executed by removing `execute` after the `Check:`

**Review by wthrowe — APPROVED** [review]
I'll convert it to a snippet in #1448.

**Comment by nilsdeppe** [comment]
All LGTM :) Thanks for being the documentation hero @gsb76!

**Pull request merged**